doc/content/api-reference/html.rst

```diff
@@ -827,6 +827,73 @@ select is no multi select, all other options get unselected automatically.
     |style             |(Dict) contains all styling attributes
 
 
+RadioGroup and RadioButton
+++++++++++++++++++++++++++
+
+.. code-block:: python
+
+    from lona.html import RadioGroup, RadioButton, Label
+
+    radio_group = RadioGroup(
+        Label('Option 1', RadioButton(value=1)),
+        Label('Option 2', RadioButton(value=2.0)),
+        Label('Option 3', RadioButton(value='3'), checked=True),
```

Comment:
`checked=True` is an argument for the `RadioButton`,  not the `Label`.

Reply:
Good catch! Fixed.